ui/src/pages/backoffice/Loan/LoanDetails/components/LoanHeader/LoanHeader.js

```diff
@@ -0,0 +1,70 @@
+import { toShortDate } from '@api/date';
+import { CopyButton, DocumentAuthors } from '@components';
+import { DocumentTitle } from '@components/Document';
+import {
+  DocumentDetailsLink,
+  LoanIcon,
+  PatronIcon,
+} from '@pages/backoffice/components';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Header, Label } from 'semantic-ui-react';
+
+export default class LoanHeader extends Component {
+  render() {
+    const { data } = this.props;
+    const recordInfo = (
+      <>
+        <label className="muted">Loan</label> {data.metadata.pid}{' '}
+        <CopyButton text={data.metadata.pid} />
+        <br />
+        <label className="muted">Created on</label> {toShortDate(data.created)}
+      </>
+    );
+    return (
+      <Grid columns={3}>
+        <Grid.Column width={6}>
+          <Header as="h1">
+            <LoanIcon />
+            <Header.Content>
+              <Header.Subheader>
+                <PatronIcon />
+                {data.metadata.patron.name}
+              </Header.Subheader>
+              Loan #{data.metadata.pid}
+              <Header.Subheader>
+                on:{' '}
+                <DocumentDetailsLink pidValue={data.metadata.document_pid}>
+                  {' '}
+                  <DocumentTitle metadata={data.metadata.document} />
```

Comment:
yeah I saw that at some point there were some changes in the component and we should align maybe how to truncate and when, because there are 2 different ways